lib/cretonne/src/ir/dfg.rs

```diff
@@ -243,6 +243,41 @@ impl DataFlowGraph {
         self.values[dest] = ValueData::Alias { ty, original };
     }
 
+    /// Replace the values of one instruction with aliases to the values of another.
```

Comment:
I usually use the term "results" for the values produced by an instruction. Or "result values".